cmd/globals.go

```diff
@@ -158,9 +158,12 @@ type serverCtxt struct {
 	ShutdownTimeout   time.Duration
 	IdleTimeout       time.Duration
 	ReadHeaderTimeout time.Duration
-
 	// The layout of disks as interpreted
 	Layout disksLayout
+	// runtime param
+	poolsRWMutex      sync.RWMutex
+	EnableExpandPools bool
```

Comment:
The `bool` type already implies "Enable"

```suggestion
	ExpandPools bool
```